src/sentry/integrations/jira/integration.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import
+
+
+from sentry.integrations import Integration, IntegrationMetadata
+
+
+metadata = IntegrationMetadata(
+    description='Sync Sentry and JIRA issues. This integration must be installed from the JIRA app store.',
+    author='The Sentry Team',
+    issue_url='https://github.com/getsentry/sentry/issues/new?title=JIRA%20Integration:%20&labels=Component%3A%20Integrations',
+    source_url='https://github.com/getsentry/sentry/tree/master/src/sentry/integrations/jira',
+    aspects={},
```

Comment:
It would be nice to have an aspect that renders on the frontend that basically links them to the installation page (if we can) or overrides the button or something along those lines.

Reply:
i'll add an aspect, but i don't think we're going to have a link until it's actually listed in their marketplace